### PR TITLE
cli: add quotes to Command args

### LINKED ISSUE #1000: devnet deploy error: unrecognized signer source
I created setup project using `anchor init testproject --javascript` in `$HOME/solana project` directory.
After:
1. setting up to devnet using `solana config set --url devnet`
2. changing localnet to devnet in `Anchor.toml`
3. changing program id in `lib.rs` and `Anchor.toml` file same as `solana address -k target/deploy/testproject-keypair.json`
4. `anchor build` --> success
5. `anchor deploy` --> error

anchor deploy error:
```
Deploying workspace: https://api.devnet.solana.com
Upgrade authority: $HOME/.config/solana/id.json
Deploying program "testproject"...
Program path: $HOME/solana project/target/deploy/testproject.so...
error: Invalid value for '--program-id <PROGRAM_ID>': unrecognized signer source
There was a problem deploying: Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "" }.
```

I have followed same steps as above but created project in `$HOME/solana_project` directory, `anchor deploy` worked properly.

Observation:
project path containing space could have caused the whole issue.

Analysis:
In the `solana program deploy....` command [here](https://github.com/project-serum/anchor/blob/ac3fbe8d386daedd8ed90dc2be452b2e9db54619/cli/src/lib.rs#L1842-L1855) 
for 2nd project command will be `solana program deploy ... --program-id ... $HOME/solana_project/...`
for 1st project command will be `solana program deploy ... --program-id ... $HOME/solana project/...`
a space in the whole command is causing the issue similar to how a copy/move command errors out in linux

### PR DESCRIPTION
**Pull request by fanatid**
Resolve #1000 

Can you test these changes? @shivam-agarwal-0 @vit0rr

### CODE REVIEW

**Comment by armaniferrante** [comment]
LGTM though I'll wait on feedback before merging.

**Review by vitorsouzaalmeida — APPROVED** [review]
tumblr_mocxoucTKW1s4dyyio1_400

**Review by vitorsouzaalmeida — APPROVED** [review]
tumblr_mocxoucTKW1s4dyyio1_400

for me, ok

**Pull request merged**
